.circleci/config.yml

```diff
@@ -24,3 +26,19 @@ jobs:
           name: Upload Coverage
           when: on_success
           command: bash <(curl -s https://codecov.io/bash) -Z -C $CIRCLE_SHA1
+workflows:
+  build-and-test:
+    jobs:
+      - build
+      - ship/java-publish:
+          prefix-tag: false
+          publish-command: ./gradlew clean assemble publishAndroidLibraryPublicationToMavenRepository -PossrhUsername="$OSSR_USERNAME" -PossrhPassword="$OSSR_PASSWORD" -PisSnapshot=false
```

Comment:
Currently overwriting the publish command from here, if this appears to work, we can move this inside the orb and avoid passing this entire command.

Reply:
You would probably want to keep it here anyway as I believe the command is slightly different from normal Java publishing?

Reply:
Yeah, makes sense I believe.